tiktokdl/tiktokdl.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import json
+import bs4
+import sys
+import io
+from dataclasses import dataclass
+
+
+@dataclass
+class TikTokAuthor:
+    id: int
+    unique_id: str
+    nickname: str
+    avatar: str
+
+@dataclass
+class TikTokVideo:
+    id: int
+    origin_link: str
```

Comment:
don't call this origin link, as origin can refer to a CDN origin. 
```suggestion
    permalink: str
```